charts/prometheus-druid-exporter/values.yaml

```diff
@@ -3,19 +3,42 @@ name: druid-exporter
 
 image:
   name: quay.io/opstree/druid-exporter
-  tag: v0.8
+  tag: v0.11
   pullPolicy: IfNotPresent
 
 annotations: {}
 
-podAnnotations: {}
```

Comment:
why is this deleted?

Reply:
Did re-add those and added resources to the deployment that were mentioned in the values.yaml but not used.